### PR TITLE
Merge 'dev-ur' into 'main' (first Urdu localization live version)

### PR DESCRIPTION
**Pull request by waleed318**
Merge Urdu development branch into main.

Requirements to go live have been completed.

Check list to make localization live.

- [x] validate all files which are not in the localization directory (content/ur/*)
- [x] check i18n/ur.toml exist
- [x] check l10n configuration in config.toml
- [x] acknowledge to initial localization contributors list
- [x] check Preview works

### CODE REVIEW

File: content/ur/_index.md
Comment (seokho-son) [inline content/ur/_index.md:42]: Hi @waleed318 @Saim-Safdar 

You can add ack for localization contributors. :)

for instance, 
(please note that the example is just a machine translation of Korean version)

```suggestion

The Korean localization of the Cloud Native Terminology was contributed by [Jihoon Seo](https://www.linkedin.com/in/jihoon-seo/) , [Seokho Son](https://www.linkedin.com/in/seokho-son/) , [Yunkon Kim](https://www.linkedin.com/in/%EC%9C%A4%EA%B3%A4-%EA%B9%80-47257914a/) and various contributors , including the Cloud Native Terminology Koreanization Team ([#glossary-localization-korean](https://cloud-native.slack.com/archives/C02PC0MLQKU)). 

```

**Review by seokho-son — COMMENTED** [review]
You can add acknowledgement for initial localization contributors list (add localization acknowledge after maintainers acknowledge section in the index page)

:)

Please open another PR (target: dev-ur) to add the ack.

File: content/ur/_index.md
Comment (waleed318) [reply content/ur/_index.md:42]: I'll take a look

Comment (Saim-Safdar) [reply content/ur/_index.md:42]: Thanks for pointing this out, let us add in the required info.

File: content/ur/_TEMPLATE.md
Comment (seokho-son) [inline content/ur/_TEMPLATE.md:3]:
```suggestion
status: Feedback Appreciated
```
nit: we can hide the template itself from sidebar item by changing the status.

**Review by seokho-son — APPROVED** [review]
The minor nit can be applied by another PR.

LGTM!

**Comment by seokho-son** [comment]
- [x] validate all files which are not in the localization directory (content/ur/*)
- [x] check i18n/ur.toml exist and localized
- [x] check l10n configuration in config.toml
- [x] acknowledge to initial localization contributors list (add localization acknowledge after maintainers acknowledge section in the index page)
- [x] check build and Preview works

Thanks to the Urdu l10n team! Nice work!
LGTM

This PR requires an additional formal approval from one of the maintainers.
However, since obtaining an approval in a timely manner (KubeCon) may be difficult, I have obtained consent from @jihoon-seo (currently unable to use a computer for a few days). As a maintainer, let me merge this PR.

**Pull request merged**
